src/levelEditor/preview/LevelPreviewEntities.js

```diff
@@ -53,11 +54,38 @@ function LevelPreviewEntities(props: Props): React$Node {
 		}
 
 		return levelDecos.map((dec, index) => {
+			const urlPrefix = process.env.REACT_APP_SPRITES_URL_PREFIX;
+			const originx = spritesData[dec.spr].originx;
+			const originy = spritesData[dec.spr].originy;
+
+			let decoElement;
+			if (urlPrefix == null) {
+				// Add sprite toggle here
+				decoElement = <>{dec.spr}</>;
+			} else {
+				const src = urlPrefix + dec.spr + '.png';
+				decoElement = (
+					<img
+						alt={dec.spr}
+						key={index}
```

Comment:
When switching between levels, loading new images will 'lag' as browsers don't immediately unload the image
https://stackoverflow.com/a/61023057
```suggestion
						key={src}
```